sds_in_a_box/SDSCode/upload_api.py

```diff
@@ -0,0 +1,117 @@
+import json
+import os
+import requests
+import time
+import boto3
+from jose import jwk, jwt
+from jose.utils import base64url_decode
+import logging
+
+logger = logging.getLogger()
+logging.basicConfig()
+logger.setLevel(logging.INFO)
+
+def _load_allowed_filenames():
```

Comment:
Should we create a utils folder to store these commonly used function? And explain or give example of what is/should be in `config.json`

Reply:
Yeah I was thinking someone might say we should start up a utilities module to the code, I was just being lazy for this commit and copy+pasted some things from the other lambda functions lol

Reply:
Sure if we can do that for this ticket that would be great, I'd prefer not to make major changes on this ticket just because I've got like 7 branches of the code at the moment lol